web_client/app/index.html

```diff
@@ -11,8 +11,12 @@
 
     <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,300i,400,600" rel="stylesheet">
 
-    <!-- TODO don't load if unneeded? -->
+    <!-- TODO: don't load if unneeded? -->
     <script src="https://cdn.polyfill.io/v2/polyfill.js?features=default,fetch,Array.prototype.find,Array.prototype.findIndex&excludes=Promise"></script>
+
+    <!-- Placeholder script for hot reload -->
+    <script type="text/javascript" src="http://localhost:3000/compiled/app.js"></script>
+
```

Comment:
We need to figure out a way to remove this line in the production build, but I think this works for now.